tests/integration/test_logcollector/test_location/test_location.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import datetime
+import os
+import sys
+import tempfile
+from shutil import rmtree
+
+import pytest
+from wazuh_testing import logcollector
+from wazuh_testing.logcollector import LOG_COLLECTOR_GLOBAL_TIMEOUT

```

Comment:
```suggestion
from wazuh_testing import logcollector
```

Reply:
Fixed in 0ab8c41819de47e71242dc1303b1ff76bab3103c